etc/hnsd.service

```diff
@@ -0,0 +1,38 @@
+[Unit]
+Description=Handshake Name Resolution
+Documentation=man:hnsd(8)
+Documentation=https://handshake-org.github.io/
+DefaultDependencies=no
+After=systemd-sysusers.service systemd-networkd.service
+Before=network.target nss-lookup.target shutdown.target
+Conflicts=shutdown.target
+Wants=nss-lookup.target
+
+[Service]
+AmbientCapabilities=CAP_SETPCAP CAP_NET_RAW CAP_NET_BIND_SERVICE
+CapabilityBoundingSet=CAP_SETPCAP CAP_NET_RAW CAP_NET_BIND_SERVICE
+EnvironmentFile=/etc/hnsd.conf
+ExecStart=/usr/bin/hnsd $HNSD_CONFIG
```

Comment:
Were you planning on removing `$HNSD_CONFIG`?

Reply:
No, its injected from `/etc/hnsd.conf`. It's a bit of hack to resemble a configuration file, but all it does is set the env var that gets passed to the executable.

This is only necessary since we don't have config parsing as of yet.

Reply:
I'd just recommend that flags be used and a sane default be put in the service file.

eg:

hnsd -r 127.0.0.1:53